src/components/Header/Header.jsx

```diff
@@ -0,0 +1,93 @@
+import { Button } from "@mui/material";
+import styled from "styled-components";
+import { IoMdContact } from "react-icons/io";
+import Input from "../Input/Input";
+import { ButtonStylesSuccsess } from "./ButtonStylesSuccsess.js";
+
+const Header = () => {
+  return (
+    <MyHeaderStyles>
+      <section className="dronocargo-header">
+        <p>Dronocargo</p>
+        <p>
+          Regina Zepeda
+          <IoMdContact className="contac-icon" />
+        </p>
+      </section>
+      <MyNavigationStyle>
+        <section className="delivery-header">
+          <p className="delivery-title">Delivery</p>
+          <p className="history-title">History</p>
+        </section>
+        <section className="search-section">
+          <Input />
+          <Button
+            variant="contained"
+            color="success"
+            style={ButtonStylesSuccsess}
+          >
+            New delivery
+          </Button>
+        </section>
+      </MyNavigationStyle>
+    </MyHeaderStyles>
+  );
+};
+
+export default Header;
+
+const MyHeaderStyles = styled.div`
+  margin: 10px 10px;
+
+  .contac-icon {
+    color: #00000040;
+    font-size: 15px;
+  }
+  .dronocargo-header {
+    display: flex;
+    position: relative;
+    height: 32px;
+  }
+  .dronocargo-header p:first-child {
+    position: absolute;
+    top: 15px;
+    left: 32px;
+    font-size: 10px;
+    font-weight: 550;
+  }
+  .dronocargo-header p:last-child {
+    display: flex;
+    align-items: center;
+    top: 15px;
+    position: absolute;
+    right: 32px;
+    font-size: 10px;
+  }
+`;
+const MyNavigationStyle = styled.div`
+  margin: 30px 20px;
+  padding-left: 0;
+  position: relative;
+  display: flex;
+  height: 20px;
+  align-items: center;
+  position: relative;
+  .delivery-header {
+    position: absolute;
+    display: flex;
+    left: 5px;
+
+    p {
+      margin: 5px;
+    }
+    & p:last-child {
+      color: #00000095;
+    }
+  }
+
+  .search-section {
+    display: flex;
+    position: absolute;
+    right: 5px;
```

Comment:
The issue that I see with this section is that it doesn't shrink and grow with the size of the viewport

```suggestion
    margin-left: auto;
```

Reply:
I know.